# HG changeset patch
# User Sergi Almacellas Abellana <sergi@koolpi.com>
Use None for empty value on reference fields

issue4898
review24271002

Index: trytond/trytond/modules/account/move.py
===================================================================

--- a/trytond/trytond/modules/account/move.py
+++ b/trytond/trytond/modules/account/move.py
@@ -207,7 +207,7 @@
         models = Model.search([
                 ('model', 'in', models),
                 ])
-        return [('', '')] + [(m.model, m.name) for m in models]
+        return [(None, '')] + [(m.model, m.name) for m in models]

     @classmethod
     def validate(cls, moves):

